Makefile

```diff
@@ -42,9 +43,18 @@ cover:
 
 coveralls:
 	@echo "+ $@"
+# Generate coverage report
+	@go test \
+		-coverprofile=${COVERPROFILE} \
+		-coverpkg=$(shell go list ${PKG}/... | xargs | sed -e 's/ /,/g') \
```

Comment:
Thanks for digging into this and finding a solution!

But, how does this command differ from the one in "cover" above? Seems like we should be able to use basically an identical command...?

Reply:
I'm not finding anything using the `cover` target so it makes sense to update that one.